CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+QuantumAnnealingAnalytics.jl Change Log
+=======================================
+
+### Staged
+- nothing
+
+### v0.0.1
+- Initial release
```

Comment:
first release will be v0.1.0, right?

Reply:
You're right, I had not double checked the Project.toml.  Fixed